src/Void.Core.Spec/BufferListSpec.fs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Void.Core.BufferListSpec
```

Comment:
Yeah sucker quit adding dead code to the project! If you are going to write the unit tests then write them, if you don't then don't!

The great thing about pair programming is that you can be rude to each other online during it...